.github/CODEOWNERS

```
@@ -42,6 +42,7 @@ airflow-core/src/airflow/ui/public/i18n/locales/ar/ @shahar1 @hussein-awala # +
 airflow-core/src/airflow/ui/public/i18n/locales/de/ @jscheffl # + @TJaniF @m1racoli
 airflow-core/src/airflow/ui/public/i18n/locales/es/ @bbovenzi # + @aoelvp94
 airflow-core/src/airflow/ui/public/i18n/locales/he/ @eladkal @shahar1 @romsharon98 # +@Dev-iL
+airflow-core/src/airflow/ui/public/i18n/locales/hu/ @jscheffl
```

Does @jscheffl speak Hungarian :) ?

---

Unfortunately not. I can be the *Translation sponsor* as committer but I would need to have some native speaker support. @majorosdonat would you take over the role as *Engaged Translator* following the i18n policy?
```suggestion
airflow-core/src/airflow/ui/public/i18n/locales/hu/ @jscheffl # +@majorosdonat
```